README.md

```diff
@@ -1,14 +1,87 @@
 # taubyte/go-seer
 
-Seer is a powerfull tool to parse and edit Yaml files.
+Seer is a powerfull tool to parse and edit Yaml files in a folder as if they represent one structure.
```

Comment:
Go-seer is a tool to parse and edit YAML files in a directory as one structure.

if you want to add powerful, powerful has one L